### DISCUSSION TITLE
Full example docker compose.

### DISCUSSION THREAD

**Discussion by mehmetuken** [directus/directus#4087] [Archive]
Postgresql
Directus
Nuxt or Next Frontend
Nginx
Redis

It's very helpfull example.

**Comment by benhaynes** [comment]
@WoLfulus — thoughts on this?

**Comment by rijkvanzanten** [comment]
https://docs.directus.io/guides/installation/docker/#docker-compose already includes 4/5ths of that list 🙂  I'm not sure if Next/Nuxt should be included OOTB, as that's a very opinionated front-end part that's not required to use Directus 🤔

**Reply by mehmetuken** [reply]
I think just an example. Maybe blog post or similar thing.

**Reply by alexvdvalk** [reply]
Updated link to docs can be found here: https://docs.directus.io/self-hosted/installation/docker.html#docker-compose
